### PR TITLE
D8UN-1652

### PR DESCRIPTION
**Pull request by DuttonMa**
Allow PlatformSH project disk size and database disk size to be set at project level.

### CODE REVIEW

File: src/Commands/ProjectCreateCommand.php [hunk 1]
```diff
@@ -89,6 +91,24 @@ protected function execute(InputInterface $input, OutputInterface $output): int
       }
     }
 
+    $project_disk_size = $input->getArgument('project_disk_size');
+
+    if (empty($project_disk_size)) {
+      $project_disk_size = $io->ask('Please provide a PlatformSH disk size for the project (default 8 GB)');
```
Comment (omahm) [inline src/Commands/ProjectCreateCommand.php]: I'd suggest the input value required, i.e. Please provide a PlatformSH disk size in megabytes for the project (default 8000MB)

File: src/Commands/ProjectCreateCommand.php [hunk 2]
```diff
@@ -89,6 +91,24 @@ protected function execute(InputInterface $input, OutputInterface $output): int
...
+      if (empty($project_disk_size)) {
+        $project_disk_size = 8000;
+      }
+    }
+
+    $db_disk_size = $input->getArgument('db_disk_size');
+
+    if (empty($db_disk_size)) {
+      $db_disk_size = $io->ask('Please provide a PlatformSH database disk size for the project (default 4 GB)');
```
Comment (omahm) [inline src/Commands/ProjectCreateCommand.php]: Ditto the project size request

File: src/Commands/ProjectCreateCommand.php [hunk 3]
```diff
@@ -89,6 +91,24 @@ protected function execute(InputInterface $input, OutputInterface $output): int
       }
     }
 
+    $project_disk_size = $input->getArgument('project_disk_size');
+
+    if (empty($project_disk_size)) {
+      $project_disk_size = $io->ask('Please provide a PlatformSH disk size for the project (default 8 GB)');
+      if (empty($project_disk_size)) {
```
Comment (omahm) [inline src/Commands/ProjectCreateCommand.php:98]: I'd check the input provided is an int (or empty) and not something like '8000 MB'

**Review by omahm — CHANGES_REQUESTED** [review]
Looks good, just a few suggestions around input validation

File: src/Commands/ProjectCreateCommand.php (hunk 1 above)
Comment (DuttonMa) [reply src/Commands/ProjectCreateCommand.php]: Good idea ! Have done that now

File: src/Commands/ProjectCreateCommand.php (hunk 2 above)
Comment (DuttonMa) [reply src/Commands/ProjectCreateCommand.php]: Done

File: src/Commands/ProjectCreateCommand.php (hunk 3 above)
Comment (DuttonMa) [reply src/Commands/ProjectCreateCommand.php:98]: That's done too

**Pull request merged**